tests/test_templates_broadcast.py

```diff
@@ -23,9 +23,6 @@
 from pennylane.templates import template, broadcast
 from pennylane.ops import RX, RY, Displacement, Beamsplitter, T, S, Rot, CRX, CRot, CNOT
 
```

Comment:
below contains a few improvements to the tests, which now can take flexible number of wires.